cmd/tink-server/main.go

```diff
@@ -115,6 +111,8 @@ func main() {
 		os.Exit(1)
 	}
 
+	logger.Close()
+	otelShutdown(ctx)
```

Comment:
Is it safe to exit (L111) before closing the logger and calling otelShutdown (as in the defer)?

I learned a few new tricks here https://stackoverflow.com/a/27629493/156674 (linking to the last answer which seems most fitting, but lots of tricks discussed in other answers).

Reply:
Yes, it is safe to exit. This change actually preserves the previous behavior: deferred calls are not executed when exit/panic/log.fatalf is called. It does however make the behavior more in-your-face obvious.